lib/plugins/pump.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+var _ = require('lodash');
+var moment = require('moment');
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["moment" is defined but never used](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1804879521/issues/source?bid=2452379&fileBranchId=3051223#l4)